ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Setup
+
+> Please provide the following details, the more info you can provide the
+> better.
+
+*Operating System*: <Linux | Windows | OS X>
+*Node Version*: <4.2 | 6.0 | ...>
+*web-push Version*: <1.0.0>
+
+> Please select any browsers that you are experiencing problems with:
+
+- [ ] Google Chrome
+- [ ] Firefox
+- [ ] Opera for Android
+- [ ] Samsung Internet Browser
+- [ ] Other
+
+> Have you tried different versions of these browsers? I.e. Google Chrome
+> Beta and Firefox Beta? Did they produce the same result?
```

Comment:
Can we also suggest to try with different browsers (in case the user has only tested with one)?
I'm pretty sure we would ask this anyway.